app/controllers/twitch_stream_updates_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TwitchStreamUpdatesController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def show
+    if params["hub.mode"] == "denied"
+      Rails.logger.error("Twitch Webhook was denied: #{params.to_json}")
```

Comment:
Is there something better to do than log an error here? Is there an error tracker that we might want to send this to? This is what happens when we are unable to create the webhook.